src/v3/test/e2e/tests/theming.ts

```diff
@@ -36,7 +36,7 @@ test('Theme configuration applies correctly', async (t) => {
 
   await t
     .expect(phoneAuthCoin.find('.siwFillPrimary').getStyleProperty('fill'))
-    .eql('rgb(202, 0, 228)');
+    .eql('rgb(62, 0, 70)');
```

Comment:
changed the mapping for primary color of the authcoin from dark to main, requires this test to be udpated.